runtime/doc/eval.txt

```diff
@@ -4891,7 +4892,7 @@ readfile({fname} [, {binary} [, {max}]])
 		separated with CR will result in a single long line (unless a
 		NL appears somewhere).
 		All NUL characters are replaced with a NL character.
-		When {binary} is equal to "b" binary mode is used:
+		When {binary/append} contains "b" binary mode is used:
```

Comment:
As can be seen from the original patch, the change from `binary` to `binary/append` is in another, older runtime update, not in this one.

I'd rather get runtime patches ported in chronological order to avoid such conflicts.